### PR TITLE
added link to contribution page creation instructions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/membership/online-membership-sign-up.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ contribution page for online sign up (see free memberships below for
 instructions on how to do this).
 
 Contribution pages are very powerful and have a lot of options that are
-grouped together into tabs. Once you have given your contribution page a
+grouped together into tabs; instructions for creating one are [here](https://docs.civicrm.org/user/en/latest/contributions/online-contributions/). Once you have given your contribution page a
```
Comment (homotechsual) [inline docs/membership/online-membership-sign-up.md]:
```suggestion
grouped together into tabs.
You may wish to read the [Online Contributions](https://docs.civicrm.org/user/en/latest/contributions/online-contributions/) documentations chapter, which provides instructions for setting up contribution pages. 
Once you have given your contribution page a
```

**Review by homotechsual — CHANGES_REQUESTED** [review]

File: docs/membership/online-membership-sign-up.md (hunk 1 above)
Comment (homotechsual) [reply docs/membership/online-membership-sign-up.md]: We generally try and avoid links that don't describe themselves - for accessibility reasons mainly. Avoiding links that just have the text "here" or "read" unless we're also adding separate link titles.

**Comment by homotechsual** [comment]
Thanks for this @irenemeisel :-) One small change from me - thanks for kicking this off and noticing that it could be improved!

**Comment by ghost** [comment]
Shoot—that was one of my first commits (obviously) so I didn’t realize you’d suggested the format of the change; I can get rid of my language and add the below if you’d prefer!

File: docs/membership/online-membership-sign-up.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ contribution page for online sign up (see free memberships below for
 instructions on how to do this).
 
 Contribution pages are very powerful and have a lot of options that are
-grouped together into tabs. Once you have given your contribution page a
+grouped together into tabs; instructions for creating one are in the [online contributions](https://docs.civicrm.org/user/en/latest/contributions/online-contributions/)section of the contributions chapter. Once you have given your contribution page a
```
Comment (homotechsual) [inline docs/membership/online-membership-sign-up.md]:
```suggestion
grouped together into tabs.

 Instructions for creating contribution pages are in the [online contributions](https://docs.civicrm.org/user/en/latest/contributions/online-contributions/) section of the contributions chapter. 
Once you have given your contribution page a
```

**Review by homotechsual — CHANGES_REQUESTED** [review]

**Comment by homotechsual** [comment]
Committed a compromise solution (mainly added back in some line-breaks :-) ) Thanks for the PR @irenemeisel :-)

**Pull request merged**